packages/duplicate-package-checker-webpack-plugin/package.json

```diff
@@ -0,0 +1,48 @@
+{
+  "name": "@cerner/duplicate-package-checker-webpack-plugin",
+  "version": "1.1.0",
+  "description": "Webpack plugin that warns you when multiple versions of the same package exist in a build.",
+  "main": "lib/index.js",
+  "engines": {
+    "node": ">=10"
+  },
+  "publishConfig": {
+    "access": "public"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/cerner/terra-toolkit.git",
+    "directory": "packages/duplicate-package-checker-webpack-plugin"
+  },
+  "keywords": [
+    "Cerner",
+    "Terra",
+    "duplicate-package-checker-webpack-plugin",
+    "webpack"
+  ],
+  "author": "Cerner Corporation",
+  "license": "Apache-2.0",
+  "bugs": {
+    "url": "https://github.com/cerner/terra-toolkit/issues"
+  },
+  "scripts": {
+    "compile": "babel src --out-dir lib --copy-files --ignore src",
```

Comment:
The source is in the src directory and we basically just use babel to copy it to the lib directory so that this package looks like the others in terra-toolkit.